apps/labs/posts/thoughts-on-reviewing-for-accessibility.md

```diff
@@ -0,0 +1,195 @@
+---
+title: 'Checking for accessibility: thoughts and a checklist!'
+author: isabela-presedo-floyd
+published: June 06, 2022
+category: [Accessibility, OSS Experience]
+featuredImage:
+  src: /posts/thoughts-on-reviewing-for-accessibility/jlabaccess4.png
+  alt: Illustration of checkboxes with ticks and checkmarks
```

Comment:
The `alt` field probably has same issue as title field in which colons break the yaml parser.

Reply:
Would quotes work here too?

Reply:
> Would quotes work here too?

I would think so.

Reply:
💯 whatever approach we decide to take with the title field, we should also take the same approach here

Reply:
The safest is to always "" enclose strings